src/esp/gfx/replay/PlayerCallbacks.h

```diff
@@ -37,32 +44,11 @@ struct PlayerCallbacks {
 };
 
 /**
- * @brief Factory method that instantiates PlayerCallbacks boilerplate to handle
- * scene graph nodes.
+ * @brief Factory method that instantiates PlayerCallbacks boilerplate for
+ * callbacks that handle scene graph nodes. Note that this does not provide an
+ * implementation for all callbacks.
```

Comment:
:D Well, that ... still means the user has to go to the source to see which callbacks aren't set. My intent with the documentation is to always provide enough info so the user *doesn't* need to browse the source. I'd say something like this:

``` 
 * [...] Note that you still need to provide
 * @ref PlayerCallbacks::loadAndCreateRendererInstance_ and 
 * @ref PlayerCallbacks::changeLightSetup_ yourself.
```

I assume I won't be the only one integrating a new renderer (Eric wanted this to work with arbitrary 3rd party renderers), so it'll benefit others as well.